service/awsconfig/v4/templates/cloudconfig/small.go

```diff
@@ -0,0 +1,46 @@
+package cloudconfig
+
+const Small = `
```

Comment:
Name is a bit minimalist for me! :) The package name has it but SmallCloudConfig makes more sense to me.

Reply:
I always see this from the client side, which translates to `cloudconfig.Small`. `cloudconfig.SmallCloudConfig` stutters and things like `go vet` usually complain about it.

Reply:
Good point about the stuttering. OK, I'm fine with this.